src/rpcclient.cpp

```diff
@@ -53,6 +53,7 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "getchaintips", 1 },
     { "getblockhash", 0 },
     { "getsuperblockbudget", 0 },
+    { "getsuperblockheight", 0 },
```

Comment:
This only needed when you want first param to be convertible and currently there is no params at all.